packages/env/lib/build-docker-compose-config.js

```diff
@@ -176,26 +176,41 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: 'mariadb',
 				ports: [ '3306' ],
 				environment: {
-					MYSQL_ALLOW_EMPTY_PASSWORD: 'yes',
+					MYSQL_ROOT_PASSWORD: 'password',
+					MYSQL_DATABASE: 'wordpress',
 				},
 				volumes: [ 'mysql:/var/lib/mysql' ],
 			},
+			'tests-mysql': {
```

Comment:
Not a big fan of adding an extra MySQL container. It will mean there's a lot of extra memory usage and startup time.

Reply:
How about we keep to a single container and create two databases ourselves using `wp db create`?

https://developer.wordpress.org/cli/commands/db/create/

Reply:
Could that be a follow-up task? It seems higher effort than the current approach

Reply:
Yep absolutely.

Reply:
How about using one database, and different table prefixes for dev and tests, respectively? The `wordpress` docker image supports a `WORDPRESS_TABLE_PREFIX` env variable for that purpose.